newa/cli.py

```diff
@@ -656,6 +682,21 @@ def cmd_report(ctx: CLIContext, rp_project: str, rp_url: str) -> None:
     if not jira_token:
         raise Exception('Jira URL is not configured!')
 
+    # before reporting check if all TF requests are finished
+    # unless we have previously executed 'execute' subcommand
+    if 'execute' not in sys.argv:
```

Comment:
Is there a better way to check if specific subcommand has been executed in @click? I could also store some variable to `ctx`. The current check is "short" but I am aware of its limits and I would like to replace it with something better.

Reply:
I don't have any better idea, and it seems simple enough. We could add some `set()` field to `CLIContext` & make every subcommand add its own name to it, that would be cleaner & more obvious. Click does not seem to have an easy way to reach "all previously executed subcommand".

Reply:
Thank you, I have pushed an update with such an implementation.